.azure-ci.yml

```diff
@@ -7,6 +7,7 @@ trigger:
   branches:
     include:
     - master
+    - release/3.0
 
 # TODO: add paths to exclude CI when modifying docs or stuff not affecting the build
 pr:
```

Comment:
For PRs you need to add it here instead.

Reply:
Fixed